backend/src/emissions/schema.py

```diff
@@ -886,8 +886,10 @@ def mutate(root, info, input: AnswerJoinRequestInput = None):
                                f"not the representative of the {join_request.working_group.name}.")
 
         if not input.approve:
+            requesting_user = join_request.user
             join_request.status = 'Declined'
             join_request.save()
+            return AnswerJoinRequest(success=True, requesting_user=requesting_user)
```

Comment:
not sure actually why we return the requesting_user 🤔

Reply:
It helps to communicate in the frontend. This way I can show a alert or some notifiaction that says "the request by user xy was/was not successfully answered".